.circleci/config.yml

```
@@ -99,13 +110,27 @@ jobs:
             brew install ghostscript
             curl -sL https://sentry.io/get-cli/ | bash
       - run:
-          name: Build
+          name: Setup notifications
```

Nitpick:

```suggestion
          name: Setup Slack notifications
```

Even though the `when` step is clearly at the same level as this one, this CircleCI syntax made me think that it was related to the "Setup notifications" step. Maybe making it explicit that it's about Slack would help.